JellyfinPlayer tvOS/ConnectToServerView.swift

```diff
@@ -10,14 +10,20 @@ import SwiftUI
 
 struct ConnectToServerView: View {
     @StateObject var viewModel = ConnectToServerViewModel()
+    @State
+    var username = ""
```

Comment:
Could you put the property wrappers on the same line as the variables? This should fit with the rest of the project.

Reply:
Oh, I see.
It is probably modified when saving because of the SwiftFormat that I use personally.
Let me modify the related settings.